### ISSUE TITLE
Contrast Issues

### ISSUE THREAD

**Issue by henryjr1** [henryjr1/deeplyrooted_msu#49]
## Location: Advanced Search Page
* File: ./Display/components/DisplayBook.js
* Item: Buttons
* Button have a color contrast of 1.82. It should have a 4.5:1

## Location: Advanced Search Page
* File: ./Display/components/DisplayBook.js
* Item: Pagination
* Contrast ratio b/w number and background color behind number is not large enough.
* Fix: Change numbers color to black  

## Location: Component View
* File: ./Display/components/DisplayBook.js
* Item: Body
* Body has a color contrast of 3.97. It should have a 4.5:1

## Location: Table View
* File: ./Display/components/DisplayBook.js
* Item: Table
* Table has color contrasts of 3.97 (expected to have 4.5:1)

## Location: Table View
* File: ./Display/components/DisplayBook.js
* Item: Body
* Body has color contrasts of 3.56 (expected to have 4.5:1)

**Comment by ajm712** [comment]
Some pagination issues fixed. The highlighting of the number on the bottom bar has yet to be fixed.

**Comment by ajm712** [comment]
Fixed table view, body, and buttons
